### PR TITLE
Add mutation support for Compose

### PR DESCRIPTION
**Pull request by chao2zhang**
We already have `ApplyChangeToAndroidLayoutFileMutator` to apply changes to the UI layer represented by XML files on Android. As Compose becomes stable, we should have `ApplyChangeToComposableKotlinFileMutator` to provide ways to apply ABI changes to the UI layer written in Compose as well.

### CODE REVIEW

**Comment by chao2zhang** [comment]
It does not seem that I could add labels. @JLLeitschuh would you mind labeling this PR?

**Comment by JLLeitschuh** [comment]
I'm actually not certain what labels are required here. @lptr @wolfs ?

**Comment by chao2zhang** [comment]
My bad - Enforce-label action was upgraded in https://github.com/gradle/gradle-profiler/pull/359, but somehow this is not triggered until @JLLeitschuh 's change today.

**Comment by lptr** [comment]
> I'm actually not certain what labels are required here. @lptr @wolfs ?

If you click on `Details`, it says:

```text
Error: Select at least one label ['bug','enhancement','internal','no-changelog'] for the PR
```

I added `enhancement`.

**Comment by chao2zhang** [comment]
Since this PR has been open for 2 weeks, would you mind reviewing it?

**Review by eskatos — COMMENTED** [review]
Thank you @chao2zhang for your contribution. This looks good!

I think `apply-kotlin-composable-change-to` would be a better name, more explicit.

File: README.md [hunk 1]
```diff
@@ -270,6 +270,7 @@ A scenario can define changes that should be applied to the source before each b
 - `apply-android-resource-value-change-to`: Change a string resource in an Android resource file.
 - `apply-android-manifest-change-to`: Add a permission to an Android manifest file.
 - `apply-android-layout-change-to`: Add a hidden view with id to an Android layout file. Supports traditional layouts as well as Databinding layouts with a ViewGroup as the root element.
+- `apply-composable-change-to`: Add a @Composable function to a Kotlin source file.
```
[the remark this answers is not in the document]
Comment (chao2zhang) [reply README.md]: Addressed!

**Comment by eskatos** [comment]
Thanks again Chao!
This will be included in the next `gradle-profiler` release

**Pull request merged**